patch/cavm-as7512/driver-i2c-muxes-pca954x-always-deselect.patch

```diff
@@ -0,0 +1,23 @@
+Always deselect channel after transaction for PCA954x.
```

Comment:
Can you explain the nature of this patch, is it a driver bug we are fixing (upstreamed) or is it a workaround for the hardware platform.